PA.py

```diff
@@ -68,6 +71,40 @@ def get_command():
         return query
 
 
+def screen_record():                                                                   #Screen Recording function              
```

Comment:
add this function in a separate file

Reply:
@Umesh-01 i have moved the function to a separate file please do check and see if it's okay :)...